Library/Homebrew/cmd/test-bot.rb

```diff
@@ -768,6 +768,9 @@ def run
   end
 
   def test_ci_upload(tap)
+    require "formulary"
+    Formulary.class_eval { define_method(:load_formula) { raise "Loaded formula!" } }
```

Comment:
A good way to test it is adding code like `File.write` in the first line of formula.